### ISSUE TITLE
Document how container with persistent data should be run

### ISSUE THREAD

**Issue by adelton** [SatelliteQE/GreenTea#40]
Checking the https://github.com/SatelliteQE/GreenTea/wiki I fail to figure out how the GreenTea container should be started so that the data is stored persistently, somewhere. Please add documentation describing that use-case.

**Comment by adelton** [comment]
Any chance of making progress with this? Does 571115a038d03bde365c7ba4a7b38c2aead9d3a1 mean that everything that comprises a GreenTea instance is now stored somewhere (`/data`?) and thus for presistent operation, `-v /opt/greentea-data:/data:Z` should be used?

**Comment by Pajinek** [comment]
Yes, I have been working on it and you described it very well. My plans are creating of storage that comprises all files created by GreenTea. You only need to set value STORAGE_ROOT in settings, you can use same path  for Docker as data volume. I haven't tested it with docker yet.
